contracts/contribute/ContributeManager.sol

```diff
@@ -39,12 +39,6 @@ error WrongStakingInstance(address stakingInstance);
 /// @param multisig Multisig address.
 error WrongServiceSetup(uint256 socialId, uint256 serviceId, address multisig);
 
-/// @dev Wrong service state.
```

Comment:
Reverted to the previous faulty version for now, as the new one went to `Contributors` completely.